tests/it/main.rs

```diff
@@ -1,5 +1,5 @@
 use coreos_stream_metadata::Stream;
-use coreos_stream_metadata::{fcos, AwsRegionImage};
+use coreos_stream_metadata::{fcos, RegionImage};
```

Comment:
one nit: It looks slightly weird that this is a `RegionImage` since kubevirt does not have that concept.

Reply:
Yeah, it would probably make sense to rename this to `ImageRelease` or so.

Reply:
@cgwalters This naming matches the existing structs in stream-metadata-go (xref https://github.com/coreos/stream-metadata-go/pull/44).  Do you want us to rename there too?

Reply:
Yes?  I mean, this isn't a strongly held opinion, and we have bigger/more interesting problems to solve, but...if we're planning to do any other semver bumps this seems like a good one to roll in.